src/components/molecules/DashboardItem.tsx

```diff
@@ -35,7 +35,7 @@ const DashboardItem: React.FC<DashboardItemProps> = ({
                     alignSelf="center"
                     marginTop={10}
                 >
-                    <Paragraph width="15%" height="100%" size={textSize}>
+                    <Paragraph width="15%" height="100%" size={textSize || 300}>
```

Comment:
You can set this in the prop list instead to make it _a proper_ default value.

Reply:
`textSize = 300,`

Reply:
On it:)